Airports/EGPE/Runway.txt

```diff
@@ -1,2 +1,2 @@
-05  23  053 233 N057.32.14.550 W004.03.31.240 N057.32.37.000 W004.02.42.650
-11  31  113 293 N057.32.40.710 W004.03.35.530 N057.32.33.090 W004.02.55.720
\ No newline at end of file
+05  23  049 229 N057.32.14.550 W004.03.31.240 N057.32.52.810 W004.02.08.050
+11  29  109 289 N057.32.40.710 W004.03.35.530 N057.32.33.090 W004.02.55.720
```

Comment:
Source for the (new) runway headings? Doesn't seem to match what I have in the AIP.